config/initializers/themes/conferences_primer_theme.rb

```diff
@@ -221,5 +229,5 @@
     label: 'Footer'
   }]
 
-  theme.plugins = ['conferences']
+  theme.plugins = ['conferences', 'journal']
```

Comment:
```suggestion
  theme.plugins = %w[conferences journal]
```
(I (Rubocop) am not strict about this if there's only one element)